### PR TITLE
Fix typos in SE-0253.

### PR DESCRIPTION
**Pull request by dan-zheng**
Remove duplicated text.

### CODE REVIEW

**Comment by dan-zheng** [comment]
cc @rxwei @lattner

**Comment by rxwei** [comment]
There's a bunch of non-ASCII "..."s. Can you fix those as well?

**Comment by dan-zheng** [comment]
> There's a bunch of non-ASCII "..."s. Can you fix those as well?

Done.

**Comment by rxwei** [comment]
Also make a link on line 7 with text "apple/swift#23517".

**Comment by dan-zheng** [comment]
> Also make a link on line 7 with text "[apple/swift#23517](https://github.com/apple/swift/pull/23517)".

Done.

File: proposals/0253-callable.md [hunk 1]
```diff
@@ -326,15 +325,15 @@ When type-checking fails, error messages look like those for function calls. Whe
 
 ```swift
 add1("foo")
-// error: cannot invoke ‘add1’ with an argument list of type '(String)'
+// error: cannot invoke ‘add1' with an argument list of type '(String)'
```
Comment (rxwei) [inline proposals/0253-callable.md]:
```suggestion
// error: cannot invoke 'add1' with an argument list of type '(String)'
```

**Pull request merged**